portal/src/util/random.ts

```diff
@@ -0,0 +1,10 @@
+export function genRandomHexadecimalString(length: number = 16): string {
+  let result = "";
+  if (length < 0) {
```

Comment:
This condition is redundant.

Reply:
so the user is responsible to input positive length? Let's remove the condition

Reply:
I mean if the length is negative, then loop would not be entered and the result is the same, so there's no need to check it.